sites/partners/cypress/fixtures/jurisdictionalAdmin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "email": "alameda-admin@example.com",
```

Comment:
this user is added, but is assigned no listings, which isn't a state we can ever get into in the ui since assigning at least one listing is required - can we add that?

Reply:
A flow I'm seeing is that this user when they log in, no terms load - maybe because they have no jurisdiction assigned? Not sure what is happening there